RomM/api.py

```diff
@@ -203,6 +203,7 @@ def _fetch_platform_icon(self, platform_slug) -> None:
         self.status.valid_credentials = True
 
     def fetch_platforms(self) -> None:
+        platform_maps.init_env_maps()
```

Comment:
Should this happen just once on app load?

Reply:
Sure. I've moved this to main just after logging is set up. I was being dumb before and initially put it before logging and then banged my head over why my messages weren't showing up in the logs